src/locales/it.ts

```diff
@@ -0,0 +1,469 @@
+      body: `Decidete insieme quante Meraviglie utilizzare e quali includere tra quelle disponibili.`,
+      body_mapLandmark: `Decidete insieme quante Meraviglie utilizzare <i>(oltre alla Meraviglia già inclusa nella mappa)</i> e quali includere tra quelle disponibili.`,
+    },
+    chooseHirelings: {
+      title: `Preparare i Combattenti`,
+      body: `Decidete insieme se utilizzare i combattenti e quali includere tra quelli disponibili.`,
+    },
+    postHirelingSetup: {
+      subtitle: `Posizionare gli Indicatori Combattente`,
+      body: `Posizionate i tre indicatori Combattente contrassegnati con "4", "8" e "12" sugli spazi "4", "8" e "12" del tracciato del punteggio sulla mappa.`,
+    },
+    drawCards: {
+      title: `Pescare Cinque Carte`,
+      body: `Ogni giocatore pesca cinque carte dal mazzo condiviso. <i>(Successivamente sceglierete tre carte da tenere in mano.)</i>`,
+    },
+    chooseFactions: {
+      title: `Preparare le Fazioni`,
+      body: `Scegliete insieme quali fazioni volete includere nella preparazione.`,
+    },
+    selectFaction: {
+      subtitle: `Assegnare le fazioni`,
+      subtitle_useDraft: `Scegliere le fazioni`,
+      body: `Assegnate ciascuna delle fazioni elencate qui sotto ad un giocatore in qualsiasi modo preferite. Le fazioni saranno preparate in senso orario. Potete selezionare qualsiasi fazione per leggere le sue statistiche.`,
+      body_useDraft: `Giocatore {{count}}, scegli con quale fazione vuoi giocare tra quelle elencate qui sotto. Puoi selezionare qualsiasi fazione per leggere le sue statistiche, poi conferma la tua selezione premendo su "Passo Successivo".`,
+    },
+    placeScoreMarkers: {
+      title: `Posizionare gli Indicatori Punteggio`,
+      title_vagabondSetUp: `Posizionare gli Indicatori Punteggio e Relazione`,
+      body: `Ogni giocatore posiziona l'indicatore punteggio della propria fazione nello spazio "0" del tracciato del punteggio sulla mappa.`,
+      body_vagabondSetUp: `Ogni giocatore posiziona l'indicatore punteggio della propria fazione nello spazio "0" del tracciato del punteggio sulla mappa. Per ogni Vagabondo, il suo giocatore posiziona l'indicatore relazione di ogni altra fazione (non Vagabondo) nello spazio Indifferente del suo grafico Relazioni.`,
+    },
+    chooseHand: {
+      title: `Scegliere la Mano Iniziale`,
+      body: `Ogni giocatore sceglie tre carte da tenere in mano e posiziona le altre due carte a faccia in giù sul mazzo condiviso. Quando tutti hanno scelto, mescolate il mazzo condiviso.`,
+    },
+    setupEnd: {
+      title: `Cominciare a Giocare`,
+      body: `La preparazione è conclusa. La partita inizia dal Giocatore {{count}}. Se vuoi ricominciare la preparazione, usa il pulsante qua sotto.<p><i>"Root Automated Setup" Sviluppato da Ewen Cameron</i><br/><i>Basato sul gioco da tavolo "Root" pubblicato da Leder Games</i><br/><i>Applicazione concessa in licenza sotto CC BY-NC-SA 4.0</i></p>`,
```

Comment:
Looking great! Thank you for taking the time to submit a very polished PR!

Only thing I'll mention is that you can feel free to credit yourself here like the other Translators did. Otherwise this is good to merge.

Reply:
Ah, missed that! Here we go 068c4c4937d2ff3136dcf9dc2574a5c524755f29